app/test_main.py

```diff
@@ -1,3 +1,58 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+def test_zero_age() -> None:
+    assert get_human_age(0, 0) == [0, 0],\
+        f"Expected [0, 0], but got {get_human_age(0, 0)}"
```

Comment:
prefer you use braces 
```suggestion
    assert (get_human_age(0, 0) == [0, 0],
        f"Expected [0, 0], but got {get_human_age(0, 0)})"
```